### PR TITLE
Refactor some code into helper functions

### PR DESCRIPTION
**Pull request by v-lopez**
This just moves some generic code out of the PlayMotion class, so that it can be used more easily by other code.

This invalidates #2

@po1 @adolfo-rt please review, thanks!

### CODE REVIEW

**Comment by po1** [comment]
Also, changing the indentation style is fine if you have a good reason to do it, but above all it should stay consistent within the project. It is common practice to observe the following points:
1. make a separate commit for the cosmetic changes
2. favor uniformity over conformance to a specific style, if you modify one file, you should edit the other ones as well

**Comment by v-lopez** [comment]
You're right, I'm too trigger happy with the auto-indentation shortcut of my IDE. I'll see what can be done.

**Comment by v-lopez** [comment]
I've fixed the problem with the libraries and updated the style of the whole package to our coding standard.

File: CMakeLists.txt [hunk 1]
```diff
@@ -19,7 +22,7 @@ add_executable(play_motion
   src/move_joint_group.cpp
   src/controller_updater.cpp)
 
-target_link_libraries(play_motion ${catkin_LIBRARIES} ${Boost_LIBRARIES})
+target_link_libraries(play_motion play_motion_helper ${Boost_LIBRARIES})
```
Comment (po1) [inline CMakeLists.txt]: I believe `${catkin_LIBRARIES}` should be here as well.
I'd love to be proven wrong, but my initial guess is that some stuff that `play_motion` needs could not be pulled by `play_motion_helper`.

I hate overcrowding my dependencies, but here I have a bad feeling about this.

**Comment by v-lopez** [comment]
I tested play_motion with this dependency configuration and it was working properly. Nevertheless better safe than sorry.

**Comment by po1** [comment]
Looks good! thanks

**Pull request merged**